### ISSUE TITLE
error while evaluating the ingress spec: could not find service "default/cm-acme-http-solver-cjnqk"

### ISSUE THREAD

**Issue by clehene** [ahmetb/gke-letsencrypt#30]
The certificate gets issued 
```
Conditions:
    Last Transition Time:  2018-12-16T02:35:38Z
    Message:               Certificate issued successfully
```
However, the load balancer only gets a HTTP/80 frontend and has the backend in `UNKNOWN` state.

**Comment by ahmetb** [comment]
1. This is not a support forum. :)
2. This is not enough to debug. Please send your Ingress describe output too.

Have you read the Troubleshooting section?

**Comment by clehene** [comment]
Hey, thanks. I understand it's not a support forum and sorry for the lack of detail. 
This said, your document is referred to from the official gRPC / Endpoints / TLS documentation https://cloud.google.com/endpoints/docs/grpc/enabling-ssl. Given that gRPC / Endpoints is presented without any warnings of not being production ready I was surprised to see how "handwavey" the TLS part is - the entire documentation assumes h2c without being very clear about it.

Rather than getting support, I was hoping to point to areas that could be improved. 

~~I don't have the context anymore as I've deleted the cluster~~. It was `1.11` and I did several deployments / deletes etc. I should not that cert-manager was deployed through an upgrade from 0.4.1  as it's failing if attempting to install 0.5.2 directly.
After setting it up with 1.10 it did work. 

I'll try a new clean deployment and get gRPC / ESP working using this. If I have any findings / problems, I'll get back with details. 

later edit: still have the cluster - will retry the setup and report
